app/models/event.rb

```diff
@@ -17,6 +17,7 @@ class Event < ActiveRecord::Base
 
   validates :title, presence: true
   validates :starts_at, presence: true
+  has_many :members
```

Comment:
I tried to create a member from the dashboard. Once the user created, I got an error because the key is missing. 

This will be used for RSVPs? I suggest to remove that `:has_many` line and add it back once we implement the RSVP feature

<img width="1077" alt="screen shot 2015-12-09 at 10 28 39 pm" src="https://cloud.githubusercontent.com/assets/1037105/11706167/4a8390d4-9ec4-11e5-84b6-301286c938fe.png">